pkgs/applications/gis/qgis/default.nix

```diff
@@ -19,7 +19,7 @@ stdenv.mkDerivation rec {
   # To handle the lack of 'local' RPATH; required, as they call one of
   # their built binaries requiring their libs, in the build process.
   preBuild = ''
-    export LD_LIBRARY_PATH=`pwd`/output/lib:$LD_LIBRARY_PATH
+    export LD_LIBRARY_PATH=`pwd`/output/lib:${stdenv.lib.makeLibraryPath [ openssl ]}:$LD_LIBRARY_PATH
```

Comment:
Still, it's certainly not as risky as adding `LD_LIBRARY_PATH` to the wrapper &ndash; is that really needed? (That's problematic e.g. if it can spawn executables from other packages, such as a browser to show help.)

BTW, /cc maintainer @viric.